packages/airnode-node/src/evm/gas-prices.ts

```diff
@@ -1,20 +1,49 @@
 import { ethers } from 'ethers';
 import { go } from '../utils/promise-utils';
 import * as logger from '../logger';
-import { DEFAULT_RETRY_TIMEOUT_MS } from '../constants';
-import { LogsData } from '../types';
+import { BASE_FEE_MULTIPLIER, DEFAULT_RETRY_TIMEOUT_MS, PRIORITY_FEE, WEI_PER_GWEI } from '../constants';
+import { GasTarget, LogsData, PendingLog } from '../types';
 
 interface FetchOptions {
   readonly provider: ethers.providers.JsonRpcProvider;
 }
 
-export async function getGasPrice(options: FetchOptions): Promise<LogsData<ethers.BigNumber | null>> {
+export const getGasPrice = async (options: FetchOptions): Promise<LogsData<GasTarget | null>> => {
```

Comment:
Maybe we want to decide on a consistent way defining functions as a team. I find `export async function getGasPrice` _slightly_ easier to read at a glance because of the "function" keyword. Plus you get function hoisting for free, although I don't think it makes a difference for top level functions. I think @Siegrift also prefers the `expost const getGetPrice = async` approach.

This is obviously a suuuuuuper minor issue though but I do think consistency is important. I don't mind either way

Reply:
haha yeah, I'm not fussed and I totally agree @ consistency.
I think I prefer the `const` approach because it better reflects that the function is a value and I think it is also newer?